scripts/neuron-pipe/resources/pctc.js

```diff
@@ -9,7 +9,17 @@ import pc from "../../testData/pc.json";
 async function fetch(resource, token, shouldUpload, resourceURL) {
   let { short, source, url, context } = resource;
   let [base, ...urlParts] = getURIPartsFromNexusURL(url);
-  let [error, docs] = await to(getResources(url, token));
+  const options = {
+    filter: JSON.stringify({
+      op: "eq",
+      path: "rdf:type",
+      value: "nsg:Configuration"
+    }),
+    context: JSON.stringify({
+      nsg: "https://bbp-nexus.epfl.ch/vocabs/bbp/neurosciencegraph/core/v0.1.0/"
```

Comment:
URL is neither staging or prod?

Reply:
Another example of where the data is living at the moment, we need to fetch it using the right context that lives there